src/utilities/index.ts

```diff
@@ -10,3 +6,5 @@ export * from './env';
 export * from './pick';
 export * from './string';
 export * from './types';
+
+export { default as orderBy } from 'lodash.orderby';
```

Comment:
this is not expected

Reply:
Just did some research and found it was caused by the upgrade of `eslint-plugin-unicorn` plugin. Since I only bumped the version number of `eslint-config` and `ts-config`, I wonder why a lot of other dependencies also got updated by the `pnpm i`.